### ISSUE TITLE
ArgumentError: cannot find element by :accessibility_id

### ISSUE THREAD

**Issue by vikramvi** [appium/ruby_lib#530]
I've latest version of appium server and arc and I'm trying to locate element with accessibilityId

But none of the below are working

```
[3] pry(main)> @se.find_element(:accessibilityId, 'More action')
ArgumentError: cannot find element by :accessibilityId
from /Users/vikram-anna/.rvm/gems/ruby-2.3.2/gems/selenium-webdriver-3.3.0/lib/selenium/webdriver/common/search_context.rb:60:in `find_element'
[4] pry(main)> @se.find_element(:AccessibilityId, 'More action')
ArgumentError: cannot find element by :AccessibilityId
from /Users/vikram-anna/.rvm/gems/ruby-2.3.2/gems/selenium-webdriver-3.3.0/lib/selenium/webdriver/common/search_context.rb:60:in `find_element'
[5] pry(main)> @se.find_element(:accessibility_id, 'More action')
ArgumentError: cannot find element by :accessibility_id
from /Users/vikram-anna/.rvm/gems/ruby-2.3.2/gems/selenium-webdriver-3.3.0/lib/selenium/webdriver/common/search_context.rb:60:in `find_element'
[6] pry(main)> 
```

**Comment by KazuCocoa** [comment]
Same as https://github.com/appium/ruby_lib/issues/527

**Comment by KazuCocoa** [comment]
@vikramvi You can call `@se.find_element(:accessibility_id, 'More action')` with `appium_lib` 9.3.7.
